examples/baselines/bc/bc_rgbd.py

```diff
@@ -0,0 +1,510 @@
+        wandb_init(asdict(args))
+
+    ds = ManiSkillDataset(
+        args.demo_path,
+        device=device,
+        load_count=args.load_count,
+    )
+
+    gpu = args.sim_backend == "gpu"
+
+    if args.video:
+        os.makedirs(os.path.join(log_path, args.video_path))
+
+
+    envs = make_eval_envs(args.env, args.num_envs, control_mode, gpu)
+    obs, _ = envs.reset(seed = args.seed)
+
+    sampler = RandomSampler(ds)
+    batch_sampler = BatchSampler(sampler, args.batch_size, drop_last=True)
+    iter_sampler = IterationBasedBatchSampler(batch_sampler, args.max_timesteps)
+
+    data_loader = DataLoader(ds, batch_sampler=iter_sampler, num_workers=0)
+    actor = Actor(ds.states.shape[1], envs.single_action_space.shape[0]).to(device=device)
+
+    optimizer = optim.Adam(actor.parameters(), lr = args.lr)
+    best_sr = -1
+
+    for i, batch in enumerate(data_loader):
+        log_dict = {}
+
+        optimizer.zero_grad()
+        preds = actor(batch["rgb"], batch["depth"], batch["state"])
+        loss = F.mse_loss(preds, batch["action"])
+        loss.backward()
+        optimizer.step()
+
+        log_dict["charts/loss"] = loss.item()
+
+        if i % args.eval_freq == 0:
+            successes = []
+            el_steps = []
+            returns = []
+            success_once = np.zeros((args.num_envs,))
+            rewards = np.zeros((args.num_envs,))
+            obs, _ = envs.reset()
+
+            for _ in range(100):
+                if gpu:
+                    inpt = process_obs_gpu(obs)
+                else:
+                    inpt = process_obs(obs, device)
+
+                obs, reward, terminated, truncated, info = envs.step(actor.act(inpt))
+                if "success" in info:
+                    success_once += info["success"].cpu().numpy() if gpu else info["success"]
+                if gpu:
+                    rewards += reward.cpu().numpy()
+                else:
+                    rewards += reward
+
+                if "final_info" in info:
+                    fin_info = info["final_info"]
+                    mask = info["_final_info"]
+                    if gpu:
+                        el_steps.append(fin_info["elapsed_steps"][mask].cpu().numpy())
+
+                        if "success" in fin_info:
+                            successes.append(fin_info["success"][mask].cpu().numpy())
+
+                        mask = mask.cpu().numpy()
+
+                    else:
+                        fin_info = fin_info[mask]
+                        fin_info = {
+                            k: np.array([dic[k] for dic in fin_info]) for k in fin_info[0]
+                        }
+                        el_steps.append(fin_info["elapsed_steps"])
+                        if "success" in fin_info:
+                            successes.append(fin_info["success"])
+                    returns.append(rewards[mask])
+
+            log_dict["charts/eval_elapsed_steps"] = np.concatenate(el_steps).mean()
```

Comment:
for logged data, can you try to be as consistent as possible with the diffusion policy baseline?

Namely there should be

eval/success_once
eval/success_at_end
eval/episode_len
eval/return